Outlines/Bonding Curves.md

```diff
@@ -0,0 +1,15 @@
+Bonding Curves
+- Bonding Curves are pricing curves that mediate the supply of a token. (include longer definition & elaboration)
+
+- Bonding curves play a crucial role in token economics for a given crypto ecosystem. (include longer elaboration)
+
+- Provide some examples of dApps, Cryptocollectibles, FOMO3D, and other things that feature bonding curves.
```

Comment:
Focus (as much as possible) on dapps that are in production, and have users. Which might be hard, but let's try :)

Reply:
Nope, I think there are lots of good examples - happy to stick with oones already live